app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message="NotVaccinatedError"):

```

Comment:
It's a bit strange message because you just copied the name of the error. 
It will be shown for user like this: 
```
NotVaccinatedError: NotVaccinatedError
```
It'd better to explain when and why this error is raised